learning/katas/go/Introduction/Hello Beam/Hello Beam Test/go.mod

```diff
@@ -0,0 +1,25 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+module hello_beam_test
+
+go 1.13
+
+require (
```

Comment:
My IDE suggests that there are some missing indirect modules, i.e. x/lint, x/tools, go/tools.
Are those required to be committed?
```go
require (
	github.com/apache/beam v2.19.0+incompatible
	github.com/golang/protobuf v1.3.5 // indirect
	github.com/google/go-cmp v0.4.0 // indirect
	golang.org/x/lint v0.0.0-20190313153728-d0100b6bd8b3 // indirect
	golang.org/x/tools v0.0.0-20190524140312-2c0ae7006135 // indirect
	google.golang.org/grpc v1.28.1 // indirect
	honnef.co/go/tools v0.0.0-20190523083050-ea95bdfd59fc // indirect
)
```

Reply:
I ran `go mod tidy` in both Hello Beam and Hello Beam Test as well as `go mod verify`.  I also removed my entire contents of `$HOME/pkg` and `$HOME/src` and subsequently tested on both IntelliJ and GoLand.  I was able to generate a course preview for both IntelliJ and GoLand and verify tasks.  As I understand, the go.mod is auto-generated as we code in go.  In IntelliJ/GoLand it does it for you automatically.  In vim with coc, when we run `:GoImports` (I think) it also does this.